rakelib/02-test.rake

```diff
@@ -58,7 +58,6 @@ end
 test_task_for :Derby, :desc => 'Run tests against (embedded) DerbyDB'
 test_task_for :H2, :desc => 'Run tests against H2 database engine'
 test_task_for :HSQLDB, :desc => 'Run tests against HyperSQL (Java) database'
-test_task_for :MSSQL, :driver => :jtds, :database_name => 'MS-SQL (SQLServer)'
```

Comment:
how do we run tests than?

Reply:
The tests will all run from the sqlserver codebase. We can try to set something up to run from this side but since the dependency goes from the sqlserver gem to this one, its kind of hard to have the tests run on this side. Ideally the java pieces would have all been in the sqlserver codebase but that would be another project to be able to get things compiled then.